tests/unit/models/reddit/test_subreddit.py

```diff
@@ -142,7 +142,7 @@ def test_submit_gallery__missing_path(self):
 
         with pytest.raises(TypeError) as excinfo:
             subreddit.submit_gallery(
-                "Cool title", images=[{"caption": "caption"}, {"caption": "caption2"}]
+                "Cool title", [{"caption": "caption"}, {"caption": "caption2"}]
```

Comment:
Removed parameter names here?

Reply:
For the `submit_<item>` methods on can easily assume that the first argument is the title and second is the `<item>` being submitted.